numpy/ma/core.py

```diff
@@ -5541,6 +5540,22 @@ def round(self, decimals=0, out=None):
         --------
         numpy.ndarray.round : corresponding function for ndarrays
         numpy.around : equivalent function
+        
+        Notes
+        -----
+        DeprecationWarning: `round_` is deprecated as of NumPy 1.25.0, 
+        and will be removed in NumPy 2.0. Please use `round` instead.
```

Comment:
I think you wanted to add a deprecation warning to `np.ma.round_` instead of `np.ma.MaskedArray.round`?